src/node/utils/Settings.js

```diff
@@ -784,6 +784,11 @@ exports.reloadSettings = () => {
     // using Unix socket for connectivity
     console.warn('The settings file contains an empty string ("") for the "ip" parameter. The "port" parameter will be interpreted as the path to a Unix socket to bind at.');
   }
+
+  // regenerate randomVersionString so clients get a fresh plugin-definition.json after plugin
+  // install/uninstall. This also means that most of the resources are rerequested on reload and
+  // plugin installation.
+  exports.randomVersionString = randomString(4);
```

Comment:
Move the comment, assignment, and log message from [lines 45-57](https://github.com/ether/etherpad-lite/blob/f0a6236a0db683fc4ed5eed29e04e7ee11897e1f/src/node/utils/Settings.js#L45-L57) to here:

```suggestion
  /*
   * At each start, Etherpad generates a random string and appends it as query
   * parameter to the URLs of the static assets, in order to force their reload.
   * Subsequent requests will be cached, as long as the server is not reloaded.
   *
   * For the rationale behind this choice, see
   * https://github.com/ether/etherpad-lite/pull/3958
   *
   * ACHTUNG: this may prevent caching HTTP proxies to work
   * TODO: remove the "?v=randomstring" parameter, and replace with hashed filenames instead
   */
  exports.randomVersionString = randomString(4);
  console.log(`Random string used for versioning assets: ${exports.randomVersionString}`);
```